src/Configuration/RectorConfigBuilder.php

```diff
@@ -169,6 +174,13 @@ public function __invoke(RectorConfig $rectorConfig): void
 
         $uniqueSets = array_unique($this->sets);
 
+        if ($this->isWithPhpLevelUsed && $this->isWithPhpSetsUsed) {
+            throw new InvalidConfigurationException(sprintf(
+                'Your config uses "withPhp*()" and "withPhpLevel()" methods at the same time.%sPick one of them to avoid rules conflicts.',
```

Comment:
```suggestion
                'Your config uses "withPhp*()" and "withPhpLevel()" methods at the same time.%sPick one of them to avoid rule conflicts.',
```